elixir/apps/web/lib/web/components/form_components.ex

```diff
@@ -234,7 +234,7 @@ defmodule Web.FormComponents do
         class={[
           "text-sm bg-neutral-50",
           "border border-neutral-300 text-neutral-900 rounded",
-          "block p-2.5",
```

Comment:
This was adding useless padding to the select element and seems to be from copy-pasta.